pandas/core/arrays/categorical.py

```diff
@@ -2349,9 +2349,9 @@ def _can_hold_na(self):
 
     @classmethod
     def _concat_same_type(self, to_concat):
-        from pandas.core.dtypes.concat import concat_categorical
+        from pandas.core.dtypes.concat import union_categoricals
```

Comment:
is this changed on purpose? it’s an api change

Reply:
This is inside `_concat_same_type`, so no external API change 
(and `_concat_same_type` always was intended to only handle same-dtype arrays)